### DISCUSSION TITLE
Progress and Temp size always 0 in windows

### DISCUSSION THREAD

**Discussion by ricardogfp** [swar/Swar-Chia-Plot-Manager#119] [Q&A]
I'm using the plot manager in windows and started a couple of jobs, I see the plot logs computing tables and so on and the temp storage space going up, but the progress and the temp size on the viewer remains at 0

**Comment by Salsaloon** [comment]
I found running my python manager.py view command from an administrative command prompt fixed that for me. I had this same issue.

**Comment by ricardogfp** [comment]
It didn't worked for me, I'm using chia blockchain 1.1.3

**Comment by Salsaloon** [comment]
Interesting, I just checked mine and my temp_size went back to 0 even though it was running in an administrative command prompt. Maybe it was not the admin command prompt that fixed it for me but killing the running window, starting a new command prompt window and re-running the view. Either way I definitely saw the 0's and killing and restarting it is currently fixing it for me. Not sure why or when the numbers go to 0 though.

**Reply by sirmarkwei** [reply]
I'm having the same problem (running 1.1.5). Running in Admin mode for powershell didn't have any impact, I'm noticing that my logs for the 2 out of the 3 rigs i'm running plotting on are pointed at a network share. (Example: Q:\Swarlogs) and the one rig that is working has the logs pointed locally (c:\users\admin\desktop\swar\logs). Just changed my two rigs that were showing progress at 0 and restarted manager. Hopefully this will help. will report back if it does.

**Comment by garthheff** [comment]
I get this too after updating to 1.1.3

Was reported here but closed, 
https://github.com/swar/Swar-Chia-Plot-Manager/issues/81

Also looks like this one as well,
https://github.com/swar/Swar-Chia-Plot-Manager/issues/53
